cmd/bots/internal/common/environment.go

```diff
@@ -218,7 +228,12 @@ func (tgEnv *TelegramBotEnvironment) SendAlertMessage(msg []byte) {
 		return
 	}
 
-	messageToBot, err := constructMessage(alertType, msg[1:], Html)
+	nodes, err := messaging.RequestAllNodes(tgEnv.requestType, tgEnv.responsePairType)
```

Comment:
Why do you need to request all nodes for each message? For aliases? Why they are not included in messages for bots?

Reply:
Please, create new PR to fix this issue. Resolving for now.